website/cue/reference/configuration.cue

```diff
@@ -22,6 +22,105 @@ configuration: {
 			}
 		}
 
+		enrichment_tables: {
+			common:      false
+			description: """
+				Configuration options for an [enrichment table](\(urls.enrichment_tables_concept)) to be used in a
+				[`remap`](\(urls.vector_remap_transform)) transform. Currently, only [CSV](\(urls.csv)) files are
+				supported.
+
+				For the lookup in the enrichment tables to be as performant as possible, the data is indexed according
+				to the fields that are used in the search. Note that indices can only be created for fields for which an
+				exact match is used in the condition. For range searches, an index isn't used and the enrichment table
+				drops back to a sequential scan of the data. A sequential scan shouldn't impact performance
+				significantly provided that there are only a few possible rows returned by the exact matches in the
+				condition. We don't recommend using a condition that uses only date range searches.
+				"""
+			required:    false
+			type: object: options: {
+				file: {
+					required:    true
+					description: "Configuration options for the file that provides the enrichment table."
+					type: object: options: {
+						path: {
+							description: """
+								The path of the enrichment table file. Currently, only [CSV](\(urls.csv)) files are
+								supported.
+								"""
+							warnings: [
+								"In order to be used by Vector, the enrichment table file needs read-only permissions.",
```

Comment:
I assume this meant to say "Vector needs read-access to the enrichment table file"?

Now it reads as if flipping the write-bit prevents you from using the file as an enrichment table.